dash/dash-renderer/src/TreeContainer.js

```diff
@@ -237,7 +237,7 @@ class BaseTreeContainer extends Component {
         );
     }
 
-    getComponent(_dashprivate_layout, children, loading_state, setProps) {
+    getComponent(_dashprivate_layout, children, loading_state, setProps, _dashextra_controlProps) {
```

Comment:
Of course, the example code repository address is https://github.com/insistence/feffery-antd-components/tree/dash-test, please use the dash-test branch. I encapsulate dash components based on Ant Design's form component, formItem component, and input component. In Ant Design, the formItem component injects relevant parameters such as value, onChange, etc. into its child elements. These parameters are crucial for completing the form validation function. There are six images below. The first three images are test cases completed based on dash2.16.1 version. It will be found that BaseTreeContainer received the relevant parameters injected by formItem, but these injected parameters have been lost in CheckedComponent, and the corresponding AntdInput does not have these parameters, so I am unable to complete the form validation function. The last three images are test cases completed based on the dash version of this PR. It will be found that BaseTreeContainer successfully received the parameters injected by formItem, and these parameters were not lost in CheckedComponent and AntdInput, so I can complete the form validation function.

<table>
      <tr>
           <td>
                <img src="https://github.com/plotly/dash/assets/92962165/91da5d09-6801-43ea-8048-9c77084c3f1f"/>
           </td>
           <td>
                <img src="https://github.com/plotly/dash/assets/92962165/89ec8d54-a115-4caf-aa8d-2c2cbe9b754c"/>
           </td>
           <td>
                <img src="https://github.com/plotly/dash/assets/92962165/3bab12c6-bcdb-4445-9ce1-90112d137c85"/>
           </td>
     </tr>
     <tr>
           <td>
                <img src="https://github.com/plotly/dash/assets/92962165/41c1e0c7-89da-460f-906c-e01706d78e6a"/>
           </td>
           <td>
                <img src="https://github.com/plotly/dash/assets/92962165/49f6c6df-5f28-41ed-aa62-b88b78c7ac40"/>
           </td>
           <td>
                <img src="https://github.com/plotly/dash/assets/92962165/c4a28fdf-f10f-4864-8347-5eac1464d6d0"/>
           </td>
     </tr>
</table>

Reply:
By the way, running usage.py from the example code repository above can yield the results shown in the screenshot.

Reply:
@T4rk1n In the native Ant Design (antd) FormItem component, it's capable of automatically accessing the onChange methods of a large batch of form input child components. This enables many form-related functionalities to be implemented automatically. However, in Dash, due to the incomplete transference of additional props, similar shortcut functionalities cannot operate normally.